new-docs/content/any/project/faq.md

```diff
@@ -19,7 +19,7 @@ the HTTP request, and the path supplied – as well as checks on the data access
 e.g. when your application is retrieving data from the database.
 
 For more detailed discussion on where to integrate Oso in your application
-depending on your requirements, please visit our guide, Add To Your Application.
+depending on your requirements, please visit our guide, [Add Oso to an App](getting-started/application).
```

Comment:
As the CI failure indicates, this guide doesn't exist for Go or Ruby, so we'll have to add a "Coming Soon" page for each.